test/bugsnex/notice_test.exs

```diff
@@ -56,9 +56,15 @@ defmodule Bugsnex.NoticeTest do
   end
 
   test "new notice contains device data if present in the metadata" do
-    notice = Notice.new(@exception, @stacktrace, %{device: %{osVersion: "2.1.1", hostname: "web1.internal"}})
+    notice = Notice.new(@exception, @stacktrace, %{device: %{osVersion: "2.1.1"}})
     [event] =  notice.events
-    assert event.device == %{osVersion: "2.1.1", hostname: "web1.internal"}
+    assert event.device == %{osVersion: "2.1.1", host: "myhost.local"}
+  end
+
+  test "new notice contains hostname in device data" do
+    notice = Notice.new(@exception, @stacktrace, %{})
+    [event] =  notice.events
```

Comment:
in test usually we use the `assert` macro in front here e.g. `assert [event] = notice.events` :)